easybuild/easyconfigs/t/trimAl/trimAl-1.4.1-GCCcore-9.3.0.eb

```
@@ -0,0 +1,29 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Updated by: Robert QIAO, Flinders University
+#
+easyblock = 'MakeCp'
+
+name = 'trimAl'
+version = '1.4.1'
+
+homepage = 'https://github.com/scapella/trimal'
+description = """EVB, FEP and LIE simulator."""
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+toolchainopts = {'pic': True}
+
+github_account = 'scapella'
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['cb8110ca24433f85c33797b930fa10fe833fa677825103d6e7f81dd7551b9b4e']
+
```

```suggestion
builddependencies = [('binutils', '2.35')]
```

The failure in https://github.com/easybuilders/easybuild-easyconfigs/pull/12916#issuecomment-845556186 is making sure that if `binutils` is used it is the one built by EasyBuild. This is a change since the older easyconfig this update is based on.

---

Thx @branfosj. Going in.